lib/util.js

```diff
@@ -22,40 +19,22 @@ exports.makeGetFn = (targets) => {
   }
 }
 
-exports.parseEntries = function (entries, get) {
-  const files = []
-
-  /*
-   * safePushToFile
-   *
-   * Pushes to files if entry file exists
-   *
-   * @param entry {String} file in the pattern "inpath[:outpath]"
-   */
-  const safePushToFile = (entry) => {
-    entry = entry.split(':')
-    if (fs.existsSync(entry[0])) {
-      if (entry.length === 1) {
-        entry.push(`assets/${entry[0]}`)
-      }
-      files.push(entry)
-      mkdirp.sync(path.dirname(entry[1]))
-    }
-  }
-
-  // parse user-provided entries
-  entries.forEach(safePushToFile)
-
-  // if no valid user-provided files, attempt to find logical entry point
-  if (files.length === 0) {
-    const entry = get('entry') || pkg.main || 'index.js'
-    const outfile = get('outfile') || 'assets/index.js'
-    safePushToFile(`${entry}:${outfile}`)
-    if (entry === 'index.js') {
-      safePushToFile('index.css:assets/index.css')
-    }
-  }
+function unique (a) {
+  return a.reduce(function (n, e) {
+    if (n.indexOf(e) === -1) n.push(e)
+    return n
+  }, [])
+}
 
+exports.parseEntries = function (entries, get) {
```

Comment:
While it's probably not a big deal due to the number of expected inputs, it is algorithmically less efficient to loop through all entries numerous times to perform these single operations.